oracle_server/oracle_server.go

```diff
@@ -691,18 +687,17 @@ func (os *OracleServer) tryLoadingNewPlugin(f fs.FileInfo, plugConf types.Plugin
 			return
 		}
 
+		os.logger.Info("** Replacing legacy plugin with new one: ", f.Name(), f.Mode().String())
```

Comment:
Why do we use ** in the log? Is it to put more emphasis on this specific log statement?

Reply:
> Why do we use ** in the log? Is it to put more emphasis on this specific log statement?

Yes, it put more emphasis on the testing phase, since this feature is stable, I will remove the stars.

Reply:
> Why do we use ** in the log? Is it to put more emphasis on this specific log statement?

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63